server/src/migrations/20210816092729_addDefaultRoadmapColumnToUser.ts

```diff
@@ -0,0 +1,19 @@
+import * as Knex from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.table('users', (table) => {
+    table
+      .integer('defaultRoadmapId')
+      .unsigned()
+      .references('id')
+      .inTable('roadmaps')
+      .onDelete('SET NULL')
+      .index();
+  });
+}
+
+export async function down(knex: Knex): Promise<void> {
+  return knex.schema.table('users', (table) => {
+    table.dropForeign(['defaultRoadmapId']);
```

Comment:
The actual column needs also to be dropped
```suggestion
    table.dropForeign(['defaultRoadmapId']);
    table.dropColumn('defaultRoadmapId');
```